src/azul/service/source_service.py

```diff
@@ -92,7 +91,7 @@ def list_sources(self,
     #        https://github.com/DataBiosphere/azul/issues/3094
     expiration = 60
 
-    @cached_property
+    @property
     def _dynamodb(self):
         return aws.client('dynamodb')
```

Comment:
Should we define the client within `azul.deployments.aws` and then return `aws.dynamodb` here?